ReadMe.md

```diff
@@ -0,0 +1,236 @@
+# Zig-midi
```

Comment:
Thanks for the translation, now I can actually review the content.

I don't think a description of the details of the midi format belongs in a readme for a library encoding and decoding such format. A link to external resources would be fine though. The sections `Basic`, `MIDI Message Structure` are heavy on this.

A lot of the other information would be better served as doc comments in the code itself. For example the section describing `Message.Kind` enumerations should be doc comments in the code instead. I think most other sections fall into this category.

To summarize, this is not really the way I would prefer a readme to be structured. Something along the lines of https://github.com/Hejsil/zig-clap is my the style I prefer, but I have to admit that readme has also become quite verbose.